docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -51,22 +57,19 @@ or
 
 .. note::
 
-	Currently, ``ansible-doc`` command can only parse Python modules for the module documentation. If you have module written in a different programming language other than Python, please write a documentation in Python file adjacent to module file.
-
-To use a local module only in certain playbooks:
+	Currently, the ``ansible-doc`` command can parse Python modules only for the module documentation. If you have a module written in a different programming language other than Python, please write the documentation in a Python file adjacent to the module file.
```

Comment:
Nice catch here. This section needed revision. The placement of `only` in the revised version feels awkward, though. How about this:
 ```suggestion
	Currently, the ``ansible-doc`` command can parse module documentation only from modules written in Python. If you have a module written in a programming language other than Python, please write the documentation in a Python file adjacent to the module file.
```